airflow/www/extensions/init_views.py

```diff
@@ -147,6 +147,20 @@ def init_error_handlers(app: Flask):
     app.register_error_handler(404, views.circles)
 
 
+def set_cors_headers_on_response(response):
```

Comment:
This function will run for all views, not just the API.

Reply:
Ah, sorry, my Python/Flask inexperience is showing… Would it be ideal for this to be moved to an external file and then imported into `init_api_connexion`?

Reply:
This code register this handler for each view.
```
app.after_request(set_cors_headers_on_response)
```
You should use [Blueprint.after_request](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Blueprint.after_request).  The api_ap variable contains an `flask.Blueprint` object.